### PR TITLE
Add Content-Language  HTTP header feature

### PR DESCRIPTION
**Pull request by hishamco**
Addresses #3973

I just wondering if `ContentLocalization` is a proper place!!

### CODE REVIEW

**Comment by hishamco** [comment]
Nah, seems it's multi-targeting issue .. I need to figure out ..

File: src/OrchardCore.Modules/OrchardCore.Localization/Manifest.cs [hunk 1]
```diff
@@ -9,3 +9,11 @@
     Category = "Internationalization",
     Dependencies = new[] { "OrchardCore.Settings" }
 )]
+#if NET5_0
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Localization/Manifest.cs:12]: OR GREATER ?

File: src/OrchardCore.Modules/OrchardCore.Localization/Startup.cs [hunk 2]
```diff
@@ -62,4 +62,22 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddSiteSettingsPropertyDeploymentStep<LocalizationSettings, LocalizationDeploymentStartup>(S => S["Culture settings"], S => S["Exports the culture settings."]);
         }
     }
+#if NET5_0
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Localization/Startup.cs:65]: OR GREATER ?

File: src/OrchardCore.Modules/OrchardCore.Localization/Manifest.cs (hunk 1 above)
Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Localization/Manifest.cs:12]: I tried this before I submit the PR but it doesn't work, and the code is grayed out, I think `OrGreater` is not shipped (frankly I'm not sure)

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Localization/Manifest.cs:12]: https://github.com/dotnet/designs/blob/main/accepted/2020/or-greater-defines/or-greater-defines.md

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Localization/Manifest.cs:12]: worst case you can do both NET5_0 and NET5_0_OR_GREATER

**Pull request merged**